site/content/en/docs/Guides/Best Practices/_index.md

```diff
@@ -0,0 +1,43 @@
+---
+title: "Cloud Best Practices" 
+linkTitle: "Cloud Best Practices"
+date: 2023-05-12T00:00:00Z
+weight: 9
+description: "Best practices for running Agones in production."
+---
+
+## Overview
+
+Running Agones in production takes consideration, from planning your launch to figuring
+out the best course of action for cluster and Agones upgrades. On this page, we've collected
+some general best practices. We also have cloud specific pages for:
+
+* [Google Kubernetes Engine (GKE)]({{< relref "gke.md" >}})
+
+If you are interested in submitting best practices for your cloud prodiver / on-prem, [please contribute!]({{< relref "/Contribute" >}})
+
+## Separate game servers from other workloads
```

Comment:
The title is backwards from what we had before. What we are saying is to move the agones components away from game servers. 

You could also move the game servers away by using taints and tolerations for them, but the suggestion here is showing how to move the agones components so that the game servers don't need taints and tolerations. 

Consider flipping the title to be more like what we had on the other page ("Separation of Agones from GameServer nodes").

Reply:
FWIW I actually disagree with the direction we had it. I actually had read this recommendation several times due to @gongmax but I never actually parsed it fully. Why are we separating controllers from the default pool? Why are we not separating game servers and keeping the control plane, kube-system, etc, in the default pool? This sounds backwards.

I didn't intend to flip the recommendation, but the title is a reflection of the direction I think it _should_ be. Freudian slip. :)